manageiq-operator/api/v1alpha1/manageiq_types.go

```diff
@@ -64,8 +64,7 @@ type ManageIQSpec struct {
 	// +optional
 	DatabaseVolumeCapacity string `json:"databaseVolumeCapacity,omitempty"`
 
-	// Flag to indicate if Kafka and Zookeeper should be deployed (default: false)
-	// +optional
+	// Flag to indicate if Kafka and Zookeeper should be deployed (default: true)
```

Comment:
The deprecated part was fine.  We need to keep the `+optional` otherwise the CR validation will require users to fill in that field.  See `+optional` in https://book.kubebuilder.io/reference/markers/crd-validation
```suggestion
	// Deprecated: Flag to indicate if Kafka and Zookeeper should be deployed (default: true)
	// +optional
```